fe/fe-core/src/main/java/org/apache/doris/statistics/util/StatisticsUtil.java

```diff
@@ -480,6 +481,9 @@ public static boolean statsTblAvailable() {
             return false;
         }
         if (Config.isCloudMode()) {
+            if (!((CloudSystemInfoService) Env.getCurrentSystemInfo()).availableBackendsExists()) {
```

Comment:
Is this another bug fix not related to smooth upgrade?

Reply:
yes, just discovered during testing